README-source.md

```diff
@@ -633,6 +633,41 @@ module.exports = {
 };
 ```
 
+### `bundle.rawRequest`
+***`bundle.rawRequest` is only available in the `perform` for web hooks and `getAccessToken` for oauth authentication methods***
```

Comment:
Might do `>` blockquote? Just a nit, don't really mind either way. /shrug

Reply:
I agree > looks better. I'd copied the formatting from the end of the bundle meta. So I've changed that too